ecosystem/typescript/sdk/src/indexer/codegen.yml

```diff
@@ -0,0 +1,37 @@
+overwrite: true
+documents: src/indexer/queries/**/*.graphql
+schema: https://indexer.mainnet.aptoslabs.com/v1/graphql
+generates:
+  src/indexer/generated/types.ts:
+    plugins:
+      - typescript
+      - add:
+          content:
+            - "/* istanbul ignore file */"
+            - "/* tslint:disable */ "
+            - "/* eslint-disable */"
+  src/indexer/generated/operations.ts:
+    preset: import-types-preset
+    presetConfig:
+      typesPath: ./types
+    plugins:
+      - typescript-operations
+      - add:
+          content:
+            - "/* istanbul ignore file */"
+            - "/* tslint:disable */ "
+            - "/* eslint-disable */"
+  src/indexer/generated/queries.ts:
+    preset: import-types-preset
+    presetConfig:
+      typesPath: ./operations
+    plugins:
+      - typescript-graphql-request
+      - add:
+          content:
+            - "/* istanbul ignore file */"
+            - "/* tslint:disable */ "
+            - "/* eslint-disable */"
+    config:
+      documentMode: string
+      documentVariableSuffix: ""
```

Comment:
- `documentMode` is to generate queries as `string` and not `documentNode` (which need parsing after)
- by default, generated query variable has `Document` as suffix, with `documentVariableSuffix = ""` we remove the `Document`, so for example, instead of the default `GetAccountCurrentTokensDocument` we are getting `GetAccountCurrentTokens` - just makes it easier to understand when later we use this variable as the query in `indexer.ts`